src/Facing.php

```diff
@@ -56,120 +38,92 @@ private function __construct(){
 		self::EAST
 	];
```

Comment:
To keep the new logic, i think `HORIZONTAL` constant must be removed and replace by 2 functions:
- `horizontal(): array` -> return same value likes `HORIZONTAL` constant
- `isHorizontal(): bool` -> return true if the face is horizontal else false
I await your opinion